tests/test_estimators.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import narwhals as nw
+
+from playtime.estimators import SelectCols
+
+
+@pytest.mark.parametrize("lib", ["pandas", "polars"])
+def test_select_cols(lib):
+    lib = pytest.importorskip(lib)
+
+    df = nw.from_native(lib.DataFrame({"a": [1, 2], "b": [10, 20], "c": ["x", "y"]}))
```

Comment:
Why do `nw.from_native` in the test? Shouldn't the estimator take care of it?

Reply:
Oops